interop-switch/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "interop-switch.fullname" . }}
+  labels:
+    app: {{ template "interop-switch.name" . }}
```

Comment:
We are using fullname for the app label.
Please change too: app: {{ template "interop-switch.fullname" . }}